### ISSUE TITLE
apt upgrade fails when upgrading packages with system user/groups

### ISSUE THREAD

**Issue by pdsouza** [maruos/maruos#110]
## Steps to repro

1. Run `apt update && apt upgrade` with Maru 0.6.6 on Nexus 5
2. Note the `dpkg` errors

## Manual workaround

1. Patch `/etc/adduser.conf`

```
-FIRST_SYSTEM_UID=900100
-LAST_SYSTEM_UID=900999
+FIRST_SYSTEM_UID=100
+LAST_SYSTEM_UID=999

-FIRST_SYSTEM_GID=900100
-LAST_SYSTEM_GID=900999
+FIRST_SYSTEM_GID=100
+LAST_SYSTEM_GID=999
```

2. Delete the `input` group:

```
$ sudo delgroup input
```
3. Run `apt update && apt upgrade`

## What's actually going on

`apt update && apt upgrade` fails to configure packages like `systemd` and `dbus` due to Maru's custom `adduser.conf` designating system [gu]ids outside the normal 100-999 range. This was initially done to make sure that Maru Desktop system user/groups do not collide with Android system user/groups. 

Unfortunately, during our container build process, our `adduser.conf` is not applied until after `debootstrap`, at which point systemd groups like `systemd-journal` and users like `systemd-timesync` have already been created in the normal 100-999 range. Thus, during an upgrade they throw a fit since they are not within the designated range specified by `adduser.conf`.

Fortunately, Android does not use the 100-999 range so we can just leave the system [gu]ids alone to fix this issue. I am currently testing a fix.

**Comment by pdsouza** [comment]
This should be fixed in https://github.com/maruos/blueprints/commit/5a5e4ad54789fb3c15b12232ce5e308958bd2b83. This will be available in our next release.

**Comment by pdsouza** [comment]
This fix is available in 0.6.7: https://maruos.com/docs/release-notes/#v0-6-7
